core/src/main/java/io/questdb/cairo/mv/MatViewRefreshIntervalJob.java

```diff
@@ -0,0 +1,126 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2024 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.cairo.mv;
+
+import io.questdb.cairo.CairoConfiguration;
+import io.questdb.cairo.CairoEngine;
+import io.questdb.cairo.TableToken;
+import io.questdb.log.Log;
+import io.questdb.log.LogFactory;
+import io.questdb.mp.Queue;
+import io.questdb.mp.SynchronizedJob;
+import io.questdb.std.ObjList;
+import io.questdb.std.ObjObjHashMap;
+
+/**
+ * Acts as an incremental refresh scheduler for mat views with interval refresh.
+ */
+public class MatViewRefreshIntervalJob extends SynchronizedJob {
```

Comment:
There was a discussion and the conclusion is this feature is better to rename to **timer** refresh rather than **interval** refresh. Please rename this job to `MatViewTimerRefreshTriggerJob` or similar with the word timer instead of interval. Or use Scheduler wording please.

Reply:
Renamed in 0dfc16828dc7b69f1f3cde6beb80f552ad3f5a26